### DISCUSSION TITLE
Attributes and final meta clean up

### DISCUSSION THREAD

**Discussion by Shasti001** [nftchef/art-engine#310] [Help + Q&A]
Currently the meta is written as follows

  {
    "name": "Project name",
    "description": "Project description.",
    "image": "URL",
    "dna": "#",
    "edition": # ,
    "date": #,
    "attributes": [
      {
        "trait_type": "Value",
        "value": "Vale"
      },
      {
           "trait_type": "Value",
        "value": "Vale"
      },
         "trait_type": "Value",
        "value": "Vale"
      }
    ]
  }
  
  My ask is how would you get the meta cleaned to be written like below. So removing the attributes just having the trait and value. 
  
    "Item#": {
        "trait": "value",
        "trait": "vlaue",
        "trait": "value",
        "trait": "value",
        "image": "ipfs://",
        "mediaType": "image/png",
        "name": "ItemName",
      }
    }
  }
}

**Comment by nftchef** [comment]
you would need the original metadata exported for ETH and then run a custom script to reformat it. What metadata standard is this for?
